server/routerlicious/packages/protocol-definitions/src/protocol.ts

```diff
@@ -302,7 +302,7 @@ export interface INackContent {
      * Optional Retry-After time in seconds
      * If specified, the client should wait this many seconds before retrying
      */
-    retryAfter?: number;
+    retryAfterMs?: number;
```

Comment:
we can't just change this, as it's part of the protocol. we should probably leave this as is